alfresco-transform-libreoffice/alfresco-transform-libreoffice-boot/src/test/java/org/alfresco/transformer/LibreOfficeTransformationIT.java

```diff
@@ -32,32 +32,7 @@
 import static java.util.stream.Collectors.toSet;
 import static org.alfresco.transformer.EngineClient.sendTRequest;
 import static org.alfresco.transformer.TestFileInfo.testFile;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_EXCEL;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_HTML;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_IMAGE_SVG;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_OPENDOCUMENT_GRAPHICS;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_OPENDOCUMENT_PRESENTATION;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_OPENDOCUMENT_SPREADSHEET;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_OPENDOCUMENT_TEXT;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_OPENXML_PRESENTATION;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_OPENXML_SPREADSHEET;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_OPENXML_WORDPROCESSING;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_PDF;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_PPT;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_RTF;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_TEXT_CSV;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_TSV;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_VISIO;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_VISIO_2013;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_WORD;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_WORDPERFECT;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_XML;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_OPENXML_SPREADSHEET_TEMPLATE_MACRO;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_OPENXML_SPREADSHEET_ADDIN_MACRO;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_OPENXML_PRESENTATION_SLIDESHOW;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_OPENXML_PRESENTATION_SLIDESHOW_MACRO;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_OUTLOOK_MSG;
-import static org.alfresco.transformer.util.MimetypeMap.MIMETYPE_DITA;
+import static org.alfresco.transformer.util.MimetypeMap.*;
```

Comment:
Coding standards ? I believe we always expand.

Reply:
I am guessing you mean we always give the specific importation instead of an asterisk. I will change that now

Reply:
This has been done now